Add plugin: Rich Text Editor Shortcuts

joshuawootonn opened the pull request:
# I am submitting a new Community Plugin

## Repo URL

<!--- Paste a link to your repo here for easy access -->
Link to my plugin: https://github.com/joshuawootonn/rich-text-editor-shortcuts

## Release Checklist
- [x] I have tested the plugin on
  - [ ]  Windows
  - [x]  macOS
  - [ ]  Linux
  - [ ]  Android _(if applicable)_
  - [ ]  iOS _(if applicable)_
- [ ] My GitHub release contains all required files (as individual files, not just in the source.zip / source.tar.gz)
  - [x] `main.js`
  - [x] `manifest.json`
  - [ ] `styles.css` _(optional)_
- [x] GitHub release name matches the exact version number specified in my manifest.json (_**Note:** Use the exact version number, don't include a prefix `v`_)
- [x] The `id` in my `manifest.json` matches the `id` in the `community-plugins.json` file.
- [x] My README.md describes the plugin's purpose and provides clear usage instructions.
- [x] I have read the developer policies at https://docs.obsidian.md/Developer+policies, and have assessed my plugins's adherence to these policies.
- [x] I have read the tips in https://docs.obsidian.md/Plugins/Releasing/Plugin+guidelines and have self-reviewed my plugin to avoid these common pitfalls.
- [x] I have added a license in the LICENSE file.
- [x] My project respects and is compatible with the original license of any code from other plugins that I'm using.
      I have given proper attribution to these other projects in my `README.md`.

joethei commented:
["name": "Notion RichText Shortcuts",](https://github.com/joshuawootonn/notion-richtext-shortcuts-for-obsidian/blob/ef698eb1546e9339202ef09267a3f03a58c6d406/manifest.json#L3)
Please rename your plugin, the plugin name should not contain "Notion" if it's only copying functionality.

["description": "All the richtext shortcuts you love in Notion, nown i Obsidian!",](https://github.com/joshuawootonn/notion-richtext-shortcuts-for-obsidian/blob/ef698eb1546e9339202ef09267a3f03a58c6d406/manifest.json#L6)
Typo, and please don't include "Obsidian" in the description.

joshuawootonn commented:
I 
- Renamed everything in manifest to not include "Notion"
- Fixed the Typo
- Created a new release with the new manifest and updated name / id

I also had to force push a rebase since Github "syncing" of my fork or maybe git's default pull config added a merge commit.


Thanks for the review @joethei 🙏🏼
